data/os/Debian.yaml

```diff
@@ -0,0 +1,10 @@
+---
+ca_cert::supported:             true
+ca_cert::trusted_cert_dir:      '/usr/local/share/ca-certificates'
+ca_cert::update_cmd:            'update-ca-certificates'
+ca_cert::cert_dir_group:        'staff'
+ca_cert::cert_dir_mode:         '2665'
+ca_cert::package_name:          'ca-certificates'
+ca_cert::ca::ca_file_group:     'root'
+ca_cert::ca::ca_file_mode:      '0444'
```

Comment:
Why not 0644 as default? Fresh debian container once more:
No files at all in `/usr/local/share/ca-certificates` and certificates in `/usr/share/ca-certificates` have `0644`
```
root@b3059a7e2497:/# ls -l /usr/share/ca-certificates/mozilla/
total 560
-rw-r--r--. 1 root root 2772 Mar 11  2023  ACCVRAIZ1.crt
-rw-r--r--. 1 root root 1972 Mar 11  2023  AC_RAIZ_FNMT-RCM.crt
-rw-r--r--. 1 root root  904 Mar 11  2023  AC_RAIZ_FNMT-RCM_SERVIDORES_SEGUROS.crt
```

Reply:
`/usr/local/share/ca-certificates` is the right place for certs installed by this module. See the FILES section of `man update-ca-certificates`.

I'd be fine with 0644 or 0444, but I think we often use 0444 as a way to show that the files are puppet-managed and shouldn't be manually edited.

Reply:
I was just referring to `/usr/share/ca-certificates` because that's provided by the ca-certificates package with mode `0644` as an argument for using the same when adding files to `/usr/local/share/ca-certificates` :)